src/sentry/workflow_engine/models/data_source.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import (
+    BoundedBigIntegerField,
+    DefaultFieldsModel,
+    FlexibleForeignKey,
+    region_silo_model,
+)
+
+
+@region_silo_model
+class DataSource(DefaultFieldsModel):
+    __relocation_scope__ = RelocationScope.Organization
+
+    class Type(models.IntegerChoices):
+        SNUBA_QUERY_SUBSCRIPTION = 1
+
+    organization = FlexibleForeignKey("sentry.Organization")
+    query_id = BoundedBigIntegerField()
+    type = models.SmallIntegerField(choices=Type.choices)
+
+    indexes = [models.Index(fields=("type", "query_id"))]
```

Comment:
Do you plan on querying by organization + type/query_id? You might need additional indexes.

Reply:
To be honest I'm not sure how exactly we'll end up querying this, but probably having an org based index here will help